src/test/java/com/thoughtworks/pacman/core/ActorTest.java

```diff
@@ -52,7 +52,7 @@ public void advance_shouldNotMove_whenHalted() throws Exception {
         SpacialCoordinate center = new SpacialCoordinate(10 * Tile.SIZE + 1, 5 * Tile.SIZE + Tile.SIZE / 2);
```

Comment:
Bu değişiklikler sadece varolan test kodunun kırılmaması için yaptığın düzeltmeler. Senin yaptığın değişikliğin doğru çalışıp çalışmadığını test etmiyorsun. Yeni beklenilen davranışla ilgili test yazman lazım. Örneğin bir cycle geçtiğinde level sayısına göre karakterler tahtada olmaları gereken yere gelmişler mi gelmemişler mi test edebilirsin.